config/config-api/src/com/thoughtworks/go/config/preprocessor/ConfigParamPreprocessor.java

```diff
@@ -19,18 +19,16 @@
 import com.thoughtworks.go.config.CruiseConfig;
 import com.thoughtworks.go.config.GoConfigPreprocessor;
 import com.thoughtworks.go.config.ParamsConfig;
+import com.thoughtworks.go.config.PipelineConfig;
 
 /**
  * @understands Interpolation of config parameters
  */
 public class ConfigParamPreprocessor implements GoConfigPreprocessor {
     private ClassAttributeCache.FieldCache fieldCache;
-    private ClassAttributeCache.AnnotationPresentCache annotationPresentCache;
-
 
     public ConfigParamPreprocessor() {
         fieldCache = new ClassAttributeCache.FieldCache();
-        annotationPresentCache = new ClassAttributeCache.AnnotationPresentCache();
     }
```

Comment:
The class is now lying unused in FieldCache. Might as well remove it from there.